main.js

```diff
@@ -0,0 +1,145 @@
+const $playingField = $("#playing-field")
+let frogs = new Frogs()
+let timerId = ''
+let timerDisplayId = ''
+const $frogsLeft = $("#frogsLeft")
+const $level = $("#level")
+const $timeLeft = $("#time")
+const $start = $("#start")
+$frogsLeft.text("Frogs Left")
+$level.text("Level")
+$timeLeft.text("Time")
+
+const addFrogs = function () {
+    for (let i = 0; i < frogs.getLevel(); i++) {
+        let location = getRandomLocation()
+        while (locationExists(location)) {//make sure no other frogs in location
+            location = getRandomLocation()
+        }
+        const color = getRandomColor()
+        const id = Frogs.getIdCount()
+        const frog = new Frog(id, location.x, location.y, location.size, color)
+        frogs.add(frog)
+    }
+}
+
+const removeFrog = function (id) {
+    frogs.delete(id)
+}
+
+const renderFrogs = function () {
+    $playingField.empty()
+    const source = $('#frog-template').html();
+    const template = Handlebars.compile(source);
+    const frogHTML = template({ frogs: frogs.getFrogs() });
+    $playingField.append(frogHTML)
+}
+const renderGameOver = function () {
+    $playingField.empty()
+    gameOverHTML = '<h1 id="gameOver">Game Over</h1>'
+    $playingField.append(gameOverHTML)
+}
+
+$playingField.on("click", ".frog", function () {
+    removeFrog($(this).data("id"))
+    $frogsLeft.text(`${frogs.getFrogsCount()} Frogs Left`)
+    if (frogs.getFrogsCount() === 0) {
+        startNewLevel()
+    }
+    renderFrogs()
+})
+
+$start.on("click", function () {
+    startGame()
+})
+
+const startGame = function () {
```

Comment:
You should separate file and concepts, especially when using MVC, write your render functions in a different place